velox/functions/prestosql/Sequence.cpp

```diff
@@ -141,12 +167,28 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> signatures() {
           .argumentType("bigint")
           .argumentType("bigint")
           .argumentType("bigint")
+          .build(),
+      exec::FunctionSignatureBuilder()
+          .returnType("array(date)")
+          .argumentType("date")
+          .argumentType("date")
```

Comment:
Should we add a signature that takes date, data, interval?

Reply:
Yes, that's my next step, given it would introduce another template parameter and the change could be non-trivial, so I plan to do it in the following PR

Reply:
That's fine, but why would you need another template parameter?

Reply:
Because in my understanding, we have 
sequence(bigint, bigint)
sequence(bigint, bigint, bigint)
sequence(date, date)
And the following twos are:
sequence(date, date, interval day to second)
 sequence(date, date, interval year to month)

in this case, to avoid duplicate code, we would need another template parameter to represent the third argument type

Reply:
interval day to second type is backed by bigint (int64_t), so I assume we won't need another template parameter.

> interval year to month

Do you plan to add this type?